lib/Backends/Interpreter/InterpreterNodes.cpp

```diff
@@ -1136,6 +1140,10 @@ void BoundInterpreterFunction::fwdSplatInst(const glow::SplatInst *I) {
     return T->getHandle<int8_t>().clear(quantization::quantize(val, destQ));
   }
 
+  if (k == ElemKind::BoolTy) {
+    return T->getHandle<bool>().clear(I->getValue());
```

Comment:
@Max-K-cadence Our internal linter isn't liking the implicit float -> bool cast here. Can you e.g. add an explicit `static_cast<bool>(I->getValue())`, or something similar to avoid the implicit cast?

Reply:
Done!